src/amber/cli/templates/app/config/application.cr.ecr

```diff
@@ -1,16 +1,116 @@
 require "./initializers/**"
 require "amber"
-
-# load the application_controller before controllers which depend on it
 require "../src/controllers/application_controller"
 require "../src/controllers/**"
 
-# NOTE: Settings should all be in config/environments/env.yml.
-# Anything here will overwrite all environments.
-Amber::Server.configure do |setting|
-  # Server options
-  # setting.name = "<%= display_name %> web application."
-  # setting.port = 80 # Port you wish your app to run
-  # setting.log = ::Logger.new(STDOUT)
-  # setting.log.level = ::Logger::INFO
+# About Application.cr File
+#
+# This is Amber application entry point. This file is responsible for loading
+# initializers, classes, and all application related code in order to have
+# Amber::Server boot up.
+#
+# > We recommend to not modify the order of the require since the order will
+# affect the behavior of the application.
+#
+# With `Amber::Server.configure` block you can redefine the Server configuration
+# settings and use ENVIRONMENT variables and/or values evaluated at runtime.
+#
+#
+Amber::Server.configure do |settings|
+  # Use your environment variables settings here.
+  #
+  # Name: A name that identifies this application. This is not internally 
+  # used by the framework.
+  #
+  # settings.name = "<%= display_name %> web application."
+  #
+  #
+  # Colorize Logging: specifies whether or not to use ANSI color codes
+  # when logging information. Defaults to true.
+  #
+  # settings.colorize_logging = true
+  #
+  #
+  # Secret Key Base: is used for specifying a key which allows sessions
+  # for the application to be verified against a known secure key to 
+  # prevent tampering. Applications get settings.secret_key_base 
+  # initialized to a random key present in ENV["AMBER_SECRET_KEY"] or 
+  # .amber_secret_key in this order.
+  #
+  # settings.secret_key_base= ENV["SECRET_KEY_BASE"]
+  #
+  #
+  # Host: is the application server host address or ip address. Useful for when 
+  # deploying Amber to a PAAS and likely the assigned server IP is either 
+  # known or unknown. Defaults to an environment variable HOST
+  #
+  # settings.host = ENV["HOST"]
```

Comment:
ENV overrides should probably be setup to over ride if ENV exists but otherwise use the default settings from env.ymls.

EXAMPLE: 

```cr
settings.host = ENV["HOST"] if ENV["HOST"]?
```

This way if HOST is empty it will keep the existing value instead of setting it to nil.